containers/sidebar/StorageSpaceStatus.js

```diff
@@ -1,15 +1,32 @@
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { c } from 'ttag';
-import { CircularProgress, Dropdown, Icon, generateUID, useUser, usePopperAnchor } from 'react-components';
+import {
+    CircularProgress,
+    Dropdown,
+    Icon,
+    generateUID,
+    useUser,
+    usePopperAnchor,
+    useSubscription
+} from 'react-components';
 import humanSize from 'proton-shared/lib/helpers/humanSize';
+import { getPlanName, hasLifetime } from 'proton-shared/lib/helpers/subscription';
+import { PLANS } from 'proton-shared/lib/constants';
 
 import { classnames } from '../../helpers/component';
 
-const StorageSpaceStatus = ({ children }) => {
+const { VISIONARY } = PLANS;
+
+const StorageSpaceStatus = ({ upgradeButton }) => {
     const [{ MaxSpace, UsedSpace }] = useUser();
+    const [subscription = {}] = useSubscription();
```

Comment:
`useSubscription` hook returns an `Array` with 2 parameters: `subscription` and `loadingSubscription`. You need to handle the loading state in case the subscription is not yet loaded in our models. 

Example:

``` jsx
const [subscription, loadingSubscription] = useSubscription();

if (loadingSubscription) {
   return <Loader />;
}

return <>Blablabla</>
```